src/librustc_error_codes/error_codes/E0751.md

```diff
@@ -5,8 +5,8 @@ Erroneous code example:
 ```compile_fail,E0751
 trait MyTrait {}
 impl MyTrait for i32 { }
-impl !MyTrait for i32 { }
+impl !MyTrait for i32 { } // error!
 ```
 
-Negative implementations are a promise that the trait will never be
-implemented for the given types.
+Negative implementations are a promise that the trait will never be implemented
+for the given types. Therefore, both cannot have exists at the same time.
```

Comment:
```suggestion
for the given types. Therefore, both cannot exists at the same time.
```

Reply:
have and exists feels duplicated. It should be either "both cannot exists" or "cannot have both".

Reply:
Woups, forgot to remove it.